qdrant-landing/content/articles/food-discovery-demo.md

```diff
@@ -0,0 +1,233 @@
+the user has liked at least one dish. We could theoretically use the same trick as above and negate the disliked dishes, but it would be a bit weird, as Qdrant has
+that feature already built-in, and we can call it just once to do the job. It's always better to perform the search server-side. Thus, in this case [we just call 
+the Qdrant server with a list of positive and negative examples](https://github.com/qdrant/demo-food-discovery/blob/6b49e11cfbd6412637d527cdd62fe9b9f74ac699/backend/discovery.py#L166), 
+so it can find some points which are close to the positive examples and far from the negative ones.
+
+```python
+response = client.recommend_groups(
+    settings.QDRANT_COLLECTION,
+    positive=search_query.positive,
+    negative=search_query.negative,
+    group_by=settings.GROUP_BY_FIELD,
+    limit=search_query.limit,
+)
+```
+
+From the user perspective nothing changes comparing to the previous case.
+
+### Location-based search
+
+Last but not least, location plays an important role in the food discovery process. You are definitely looking for something you can find nearby, not on the other
+side of the globe. Therefore, all the search modes accepts your current location as a filtering criteria, and you can enable it by clicking on “Find near me” icon
+in the top right. This way you can find the best pizza in your neighborhood, not in the whole world. Qdrant [geo radius filter](https://qdrant.tech/documentation/concepts/filtering/#geo-radius) is a perfect choice for this. It lets you
+filter the results by distance from a given point. 
+
+```python
+from qdrant_client import models
+
+# Create a geo radius filter
+query_filter = models.Filter(
+    must=[
+        models.FieldCondition(
+            key="cafe.location",
+            geo_radius=models.GeoRadius(
+                center=models.GeoPoint(
+                    lon=location.longitude,
+                    lat=location.latitude,
+                ),
+                radius=location.radius_km * 1000,
+            ),
+        )
+    ]
+)
+```
+
+Such a filter needs [a payload index](https://qdrant.tech/documentation/concepts/indexing/#payload-index) to work efficiently, and it was created on a collection
+we used to create the snapshot. When you import it into your instance, the index will be already there.
+
+## Using the demo
+
+The Food Discovery Demo [is available online](https://food-discovery.qdrant.tech/), but if you prefer to run it locally, you can do it with Docker. The 
+[README](https://github.com/qdrant/demo-food-discovery/blob/main/README.md) describes all the steps more in detail, but here is a quick start:
+
+```bash
+git clone git@github.com:qdrant/demo-food-discovery.git
+cd demo-food-discovery
+# Create .env file based on .env.example
+docker-compose up -d
+```
+
+The demo will be available at `http://localhost:8000`, but you won't be able to search anything until you import the snapshot into your Qdrant instance.
```

Comment:
import the snapshot into your Qdrant instance - should link to instructions for how to do this

Reply:
It has been linked to the documentation.